pkg/plugin/plugin.go

```diff
@@ -356,3 +356,33 @@ func (p *plugin) BackupEtcdCluster(ctx context.Context, oc *api.OpenShiftManaged
 	}
 	return nil
 }
+
+func (p *plugin) RunCommand(ctx context.Context, oc *api.OpenShiftManagedCluster, hostname string, command api.Command) error {
+	if !validate.IsValidAgentPoolHostname(hostname) {
+		return fmt.Errorf("invalid hostname %q", hostname)
+	}
+
+	var script string
+	switch command {
+	case api.CommandRestartNetworkManager:
+		script = "systemctl restart NetworkManager.service"
```

Comment:
nit: should we point to the absolute path of the command? not sure if Geneva commands will be affected by path changes or etc...